src/cli/package-install.js

```diff
@@ -29,14 +29,25 @@ function updatePackageFile() {
 
 exports.updatePackageFile = updatePackageFile;
 
-function npmInstallProduction() {
-	cproc.execSync('npm i --production', {
+function installAll() {
+	var prod = global.env !== 'development';
+	var command = 'npm install';
+	try {
+		var packageManager = require('nconf').get('package_manager');
```

Comment:
How is `packageManager` defined, just via environment variable or config.json?

Reply:
Either. If nconf is available and has loaded environment variables (as is normal), then an env var can be used. If not, then npm is used by default.